template/apache2.conf

```diff
@@ -30,12 +30,12 @@ Listen 80
 	Require all denied
 </Directory>
 
-<Directory /app>
+<Directory ${VOLUME_PATH}>
```

Comment:
Yeah. I'm not sure why it was on the non-webroot directory before. AllowOverride just lets .htaccess files do their thing.